pkg/assembler/backends/ent/schema/dependency.go

```diff
@@ -46,7 +46,7 @@ func (Dependency) Fields() []ent.Field {
 			Unique().
 			Immutable(),
 		field.UUID("package_id", getUUIDv7()),
-		field.UUID("dependent_package_version_id", getUUIDv7()).Optional(),
+		field.UUID("dependent_package_version_id", getUUIDv7()),
```

Comment:
currently in draft due to the issue with atlas: https://github.com/guacsec/guac/issues/2057

Reply:
issue now resolved.